### PR TITLE
ungoogled-chromium-portable: Add version 84.0.4147.125

### PR DESCRIPTION
**Pull request by ouyangwanxi**
Add persistent configuration for chromium

### CODE REVIEW

File: bucket/ungoogled-chromium-portable.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchBuild-Win64/ungoogled-chromium-$version-1_windows.7z",
+                "extract_dir": "ungoogled-chromium-$version-1_windows"
+            },
+            "32bit": {
+                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchBuild-Win64/Ungoogled-Chromium-$version-Win32.7z",
+                "extract_dir": "Ungoogled-Chromium-$version-Win32"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:48]:
```suggestion
                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchBuild-Win64/Ungoogled-Chromium-$version-2_Win32.7z",
                "extract_dir": "Ungoogled-Chromium-$version-2_Win32"
```

File: bucket/ungoogled-chromium-portable.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "persist": "User Data",
+    "checkver": {
+        "github": "https://github.com/macchrome/winchrome",
+        "regex": "v([\\d.]+)-r(?<build>\\d+)-Win64"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchBuild-Win64/ungoogled-chromium-$version-1_windows.7z",
+                "extract_dir": "ungoogled-chromium-$version-1_windows"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:44]:
```suggestion
                "url": "https://github.com/macchrome/winchrome/releases/download/v$version-r$matchBuild-Win64/ungoogled-chromium-$version-2_windows.7z",
                "extract_dir": "ungoogled-chromium-$version-2_windows"
```

File: bucket/ungoogled-chromium-portable.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "##": "Check chromium.woolyss.com for different versions of Chromium releases.",
+    "version": "83.0.4103.116",
+    "description": "Browser aiming for safer, faster, and more stable way for all users to experience the web.",
+    "homepage": "https://www.chromium.org",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v83.0.4103.116-r756066-Win64/ungoogled-chromium-83.0.4103.116-1_windows.7z",
+            "hash": "sha1:10e8e055d6dc2caeea7962e66e237c82f750b5b4",
+            "extract_dir": "ungoogled-chromium-83.0.4103.116-1_windows"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:11]:
```suggestion
            "url": "https://github.com/macchrome/winchrome/releases/download/v84.0.4147.89-r768962-Win64/ungoogled-chromium-84.0.4147.89-2_windows.7z",
            "hash": "sha1:867eb6d60b315763b9be44bb0d86f7ebc61c6208",
            "extract_dir": "ungoogled-chromium-84.0.4147.89-2_windows"
```

File: bucket/ungoogled-chromium-portable.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v83.0.4103.116-r756066-Win64/Ungoogled-Chromium-83.0.4103.116-Win32.7z",
+            "hash": "sha1:065ece39ea4d93ad1dc34e7d4829de5231122b7c",
+            "extract_dir": "Ungoogled-Chromium-83.0.4103.116-Win32"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:16]:
```suggestion
            "url": "https://github.com/macchrome/winchrome/releases/download/v84.0.4147.89-r768962-Win64/Ungoogled-Chromium-84.0.4147.89-2_Win32.7z",
            "hash": "sha1:54dfe0e0247f1da68db8087d2bd9e1bd820ea79d",
            "extract_dir": "Ungoogled-Chromium-84.0.4147.89-2_Win32"
```

File: bucket/ungoogled-chromium-portable.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        }
+    },
+    "bin": [
+        [
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:20]:
```suggestion
        [
            "chrome.exe",
            "chrome",
            "--user-data-dir=\"$dir\\User Data"
        ],
        [
```

File: bucket/ungoogled-chromium-portable.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+            "--user-data-dir=\"$dir\\User Data"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "chrome.exe",
+            "Chromium",
+            "--user-data-dir=\"$dir\\User Data"
+
+
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:32]:
```suggestion
```

File: bucket/ungoogled-chromium-portable.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
+{
+    "##": "Check chromium.woolyss.com for different versions of Chromium releases.",
+    "version": "83.0.4103.116",
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:3]:
```suggestion
    "version": "84.0.4147.89",
```

File: bucket/ungoogled-chromium-portable.json [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+            "url": "https://github.com/macchrome/winchrome/releases/download/v84.0.4147.89-r768962-Win64/Ungoogled-Chromium-84.0.4147.89-2_Win32.7z",
+            "hash": "sha1:54dfe0e0247f1da68db8087d2bd9e1bd820ea79d",
+            "extract_dir": "Ungoogled-Chromium-84.0.4147.89-2_Win32"
+        }
+    },
+    "bin": [
+        [
+            "chrome.exe",
+            "chrome",
+            "--user-data-dir=\"$dir\\User Data"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:23]:
```suggestion
            "--user-data-dir=\"$dir\\User Data\""
```

File: bucket/ungoogled-chromium-portable.json [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+        ],
+        [
+            "chrome.exe",
+            "chromium",
+            "--user-data-dir=\"$dir\\User Data"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:28]:
```suggestion
            "--user-data-dir=\"$dir\\User Data\""
```

File: bucket/ungoogled-chromium-portable.json [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+        ]
+    ],
+    "shortcuts": [
+        [
+            "chrome.exe",
+            "Chromium",
+            "--user-data-dir=\"$dir\\User Data"
```
Comment (Ash258) [inline bucket/ungoogled-chromium-portable.json:35]:
```suggestion
            "--user-data-dir=\"$dir\\User Data\""
```

**Comment by Ash258** [comment]
a238b9f73de691b12e58e924295d17b45b0528b9 

Why?

**Comment by ouyangwanxi** [comment]
I think the name chromium-portable is more appropriate. It is modified from chromium.

**Comment by Ash258** [comment]
No

**Pull request merged**